i18n/fr.yaml

```diff
@@ -0,0 +1,20 @@
+about: "À propos"
+backToTop: "Revenir en haut"
+blog: "Blog"
+changeLanguage: "Changer la langue"
+changeTheme: "Thème d'affichage"
+copyright: "© 2023 Hextra Project."
+dark: "Sombre"
+documentation: "Documentation"
+editThisPage: "Modifier cette page sur GitHub →"
+hugoDocs: "Docs Hugo ↗"
+lastUpdated: "Dernière modification"
+light: "Clair"
+more: "Plus"
+noResultsFound: "Pas de résulats trouvés"
+onThisPage: "Sur cette page"
+onThisPage: "Sur cette page"
+poweredBy: "Propulsé par Hextra"
+readMore: "Lire plus →"
+searchPlaceholder: "Rechercher..."
+showcase: "Showcase"
```

Comment:
should it be translated to "Présentation"?

Reply:
I don't think so, because Présentation is too vague, it could be anything, in French we use showcase, the meaning is not unknown, or Featured, but there's no exact translation.

Reply:
got it, thanks for explaining! merci beaucoup :)